Tella/Data/Database/UwaziDatabase.swift

```diff
@@ -15,7 +15,6 @@ extension TellaDataBase: UwaziTemplateProtocol {
             cddl(D.cId, D.integer, primaryKey: true, autoIncrement: true),
             cddl(D.cTemplateId, D.text),
             cddl(D.cEntity, D.text),
-            cddl(D.cRelationships, D.text),
```

Comment:
maybe we can keep this column `cRelationships`  on creating the table instead of creating the table and then adding the column ?